### DISCUSSION TITLE
how to make pull request mediatory for contributors github

### DISCUSSION THREAD

**Discussion by GitHub-18891** [community/community#18647] [Pull Requests]
how to make pull request mediatory for contributors GitHub

I need anyone from my contributors which I added before when he/she tries to pull any codes he should ask for approval to pass the code

**Comment by MathiasReker** [comment]
You can do this in settings of the repository. There you can add this kind of rules.

**Reply by GitHub-18891** [reply]
thank you sir but can you give me the steps but am very beginner

**Comment by Zi-Tao** [comment]
@mohamedfarrag-7d https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/creating-a-pull-request
